<!DOCTYPE html>
<html lang="en-us">

<head>

  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="generator" content="Source Themes Academic 4.5.0">

  

  
  
  
  
  
    
    
    
  
  

  <meta name="author" content="Thomas Lankiewicz">

  
  
  
    
  
  <meta name="description" content="Consortium-based approaches are a promising avenue toward efficient bioprocessing. However, many complex microbial interactions dictate community dynamics and stability that must be replicated in synthetic systems. The rumen and/or hindguts of large mammalian herbivores harbor complex communities of biomass-degrading fungi and bacteria, as well as archaea and protozoa that work collectively to degrade lignocellulose, yet the microbial interactions responsible for stability, resilience, and activity of the community remain largely uncharacterized. In this work, we demonstrate a “top-down” enrichment-based methodology for selecting a minimal but effective lignocellulose-degrading community that produces methane-rich fermentation gas (biogas). The resulting enrichment consortium produced 0.75–1.9-fold more fermentation gas at 1.4–2.1 times the rate compared to a monoculture of fungi from the enrichment. Metagenomic sequencing of the top-down enriched consortium revealed genomes encoding for functional compartmentalization of the community, spread across an anaerobic fungus (Piromyces), a bacterium (Sphaerochaeta), and two methanogenic archaea (Methanosphaera and Methanocorpusculum). Guided by the composition of the top-down enrichment, several synthetic cocultures were formed from the “bottom-up” using previously isolated fungi, Neocallimastix californiae and Anaeromyces robustus paired with the methanogen Methanobacterium bryantii. While cross-feeding occurred in synthetic co-cultures, removal of fungal metabolites by methanogens did not increase the rate of gas production or the rate of substrate deconstruction by the synthetic community relative to fungal monocultures. Metabolomic characterization verified that syntrophy was established within synthetic co-cultures, which generated methane at similar concentrations compared to the enriched consortium but lacked the temporal stability (resilience) seen in the native system. Taken together, deciphering the membership and metabolic potential of an enriched gut consortium enables the design of methanogenic synthetic co-cultures. However, differences in the growth rate and stability of enriched versus synthetic consortia underscore the difficulties in mimicking naturally occurring syntrophy in synthetic systems.">

  
  <link rel="alternate" hreflang="en-us" href="/publication/2019_acs_synbio/">

  


  
  
  
  <meta name="theme-color" content="#2962ff">
  

  
  
  
  
    
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.8.6/css/academicons.min.css" integrity="sha256-uFVgMKfistnJAfoCUQigIl+JfUaP47GrRKjf6CTPVmw=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.0/css/all.css" integrity="sha384-aOkxzJ5uQz7WBObEZcHvV5JvRW3TUc2rNPA7pe3AwnsUohiw1Vj2Rgx2KSOkF5+h" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.css" integrity="sha256-ygkqlh3CYSUri3LhQxzdcm0n1EQvH2Y+U5S2idbLtxs=" crossorigin="anonymous">

    
    
    
      
    
    
      
      
        
          <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.6/styles/github.min.css" crossorigin="anonymous" title="hl-light">
          <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.6/styles/dracula.min.css" crossorigin="anonymous" title="hl-dark" disabled>
        
      
    

    
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.2.0/leaflet.css" integrity="sha512-M2wvCLH6DSRazYeZRIm1JnYyh22purTM+FDB5CsyxtQJYeKq83arPe5wgbNmcFXGqiSH2XR8dT/fJISVA1r/zQ==" crossorigin="anonymous">
    

    

  

  
  
  
  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Montserrat:400,700%7CRoboto:400,400italic,700%7CRoboto+Mono&display=swap">
  

  
  
  
  <link rel="stylesheet" href="/css/academic.min.a3cda781a9b24f26fefd34a53547835b.css">

  

  




  


  

  <link rel="manifest" href="/index.webmanifest">
  <link rel="icon" type="image/png" href="/img/icon-32.png">
  <link rel="apple-touch-icon" type="image/png" href="/img/icon-192.png">

  <link rel="canonical" href="/publication/2019_acs_synbio/">

  
  
  
  
    
  
  
  <meta property="twitter:card" content="summary_large_image">
  
  <meta property="twitter:site" content="@LordOfFungalRot">
  <meta property="twitter:creator" content="@LordOfFungalRot">
  
  <meta property="og:site_name" content="Thomas S. Lankiewicz">
  <meta property="og:url" content="/publication/2019_acs_synbio/">
  <meta property="og:title" content="Top-down enrichment guides in formation of synthetic microbial consortia for biomass degradation | Thomas S. Lankiewicz">
  <meta property="og:description" content="Consortium-based approaches are a promising avenue toward efficient bioprocessing. However, many complex microbial interactions dictate community dynamics and stability that must be replicated in synthetic systems. The rumen and/or hindguts of large mammalian herbivores harbor complex communities of biomass-degrading fungi and bacteria, as well as archaea and protozoa that work collectively to degrade lignocellulose, yet the microbial interactions responsible for stability, resilience, and activity of the community remain largely uncharacterized. In this work, we demonstrate a “top-down” enrichment-based methodology for selecting a minimal but effective lignocellulose-degrading community that produces methane-rich fermentation gas (biogas). The resulting enrichment consortium produced 0.75–1.9-fold more fermentation gas at 1.4–2.1 times the rate compared to a monoculture of fungi from the enrichment. Metagenomic sequencing of the top-down enriched consortium revealed genomes encoding for functional compartmentalization of the community, spread across an anaerobic fungus (Piromyces), a bacterium (Sphaerochaeta), and two methanogenic archaea (Methanosphaera and Methanocorpusculum). Guided by the composition of the top-down enrichment, several synthetic cocultures were formed from the “bottom-up” using previously isolated fungi, Neocallimastix californiae and Anaeromyces robustus paired with the methanogen Methanobacterium bryantii. While cross-feeding occurred in synthetic co-cultures, removal of fungal metabolites by methanogens did not increase the rate of gas production or the rate of substrate deconstruction by the synthetic community relative to fungal monocultures. Metabolomic characterization verified that syntrophy was established within synthetic co-cultures, which generated methane at similar concentrations compared to the enriched consortium but lacked the temporal stability (resilience) seen in the native system. Taken together, deciphering the membership and metabolic potential of an enriched gut consortium enables the design of methanogenic synthetic co-cultures. However, differences in the growth rate and stability of enriched versus synthetic consortia underscore the difficulties in mimicking naturally occurring syntrophy in synthetic systems."><meta property="og:image" content="/publication/2019_acs_synbio/featured.jpg">
  <meta property="twitter:image" content="/publication/2019_acs_synbio/featured.jpg"><meta property="og:locale" content="en-us">
  
    
      <meta property="article:published_time" content="2019-08-28T00:00:00&#43;00:00">
    
    <meta property="article:modified_time" content="2019-08-28T00:00:00&#43;00:00">
  

  


    











<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "Article",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "/publication/2019_acs_synbio/"
  },
  "headline": "Top-down enrichment guides in formation of synthetic microbial consortia for biomass degradation",
  
  "image": [
    "/publication/2019_acs_synbio/featured.jpg"
  ],
  
  "datePublished": "2019-08-28T00:00:00Z",
  "dateModified": "2019-08-28T00:00:00Z",
  
  "author": {
    "@type": "Person",
    "name": "S. Gilmore"
  },
  
  "publisher": {
    "@type": "Organization",
    "name": "Thomas S. Lankiewicz",
    "logo": {
      "@type": "ImageObject",
      "url": "/img/icon-512.png"
    }
  },
  "description": "Consortium-based approaches are a promising avenue toward efficient bioprocessing. However, many complex microbial interactions dictate community dynamics and stability that must be replicated in synthetic systems. The rumen and/or hindguts of large mammalian herbivores harbor complex communities of biomass-degrading fungi and bacteria, as well as archaea and protozoa that work collectively to degrade lignocellulose, yet the microbial interactions responsible for stability, resilience, and activity of the community remain largely uncharacterized. In this work, we demonstrate a “top-down” enrichment-based methodology for selecting a minimal but effective lignocellulose-degrading community that produces methane-rich fermentation gas (biogas). The resulting enrichment consortium produced 0.75–1.9-fold more fermentation gas at 1.4–2.1 times the rate compared to a monoculture of fungi from the enrichment. Metagenomic sequencing of the top-down enriched consortium revealed genomes encoding for functional compartmentalization of the community, spread across an anaerobic fungus (Piromyces), a bacterium (Sphaerochaeta), and two methanogenic archaea (Methanosphaera and Methanocorpusculum). Guided by the composition of the top-down enrichment, several synthetic cocultures were formed from the “bottom-up” using previously isolated fungi, Neocallimastix californiae and Anaeromyces robustus paired with the methanogen Methanobacterium bryantii. While cross-feeding occurred in synthetic co-cultures, removal of fungal metabolites by methanogens did not increase the rate of gas production or the rate of substrate deconstruction by the synthetic community relative to fungal monocultures. Metabolomic characterization verified that syntrophy was established within synthetic co-cultures, which generated methane at similar concentrations compared to the enriched consortium but lacked the temporal stability (resilience) seen in the native system. Taken together, deciphering the membership and metabolic potential of an enriched gut consortium enables the design of methanogenic synthetic co-cultures. However, differences in the growth rate and stability of enriched versus synthetic consortia underscore the difficulties in mimicking naturally occurring syntrophy in synthetic systems."
}
</script>

  

  


  


  





  <title>Top-down enrichment guides in formation of synthetic microbial consortia for biomass degradation | Thomas S. Lankiewicz</title>

</head>

<body id="top" data-spy="scroll" data-offset="70" data-target="#TableOfContents" >

  <aside class="search-results" id="search">
  <div class="container">
    <section class="search-header">

      <div class="row no-gutters justify-content-between mb-3">
        <div class="col-6">
          <h1>Search</h1>
        </div>
        <div class="col-6 col-search-close">
          <a class="js-search" href="#"><i class="fas fa-times-circle text-muted" aria-hidden="true"></i></a>
        </div>
      </div>

      <div id="search-box">
        
        <input name="q" id="search-query" placeholder="Search..." autocapitalize="off"
        autocomplete="off" autocorrect="off" spellcheck="false" type="search">
        
      </div>

    </section>
    <section class="section-search-results">

      <div id="search-hits">
        
      </div>

    </section>
  </div>
</aside>


  
<nav class="navbar navbar-light fixed-top navbar-expand-lg py-0 compensate-for-scrollbar" id="navbar-main">
  <div class="container">

    
      <a class="navbar-brand" href="/">Thomas S. Lankiewicz</a>
      
      <button type="button" class="navbar-toggler" data-toggle="collapse"
              data-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
        <span><i class="fas fa-bars"></i></span>
      </button>
      

    
    <div class="collapse navbar-collapse" id="navbar">

      
      
      <ul class="navbar-nav mr-auto">
        

        

        
        
        
          
        

        
        
        
        
        
        
          
          
          
            
          
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#projects"><span>Research</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        
          
          
          
            
          
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#publications"><span>Publications</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        
          
          
          
            
          
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#contact"><span>Contact</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        

        <li class="nav-item">
          <a class="nav-link " href="/files/cv.pdf"><span>CV</span></a>
        </li>

        
        

      
      </ul>
      <ul class="navbar-nav ml-auto">
      

        

        
        <li class="nav-item">
          <a class="nav-link js-search" href="#"><i class="fas fa-search" aria-hidden="true"></i></a>
        </li>
        

        

        
        <li class="nav-item">
          <a class="nav-link js-dark-toggle" href="#"><i class="fas fa-moon" aria-hidden="true"></i></a>
        </li>
        

      </ul>

    </div>
  </div>
</nav>


  <div class="pub">

  




















  
  
    
  


<div class="article-container pt-3">
  <h1>Top-down enrichment guides in formation of synthetic microbial consortia for biomass degradation</h1>

  

  



<div class="article-metadata">

  
  
  
  
  <div>
    



  
  <span>S. Gilmore</span>, <span>T. Lankiewicz</span>, <span>S.E. Wilken</span>, <span>J. Brown</span>, <span>J. Sexton</span>, <span>J. Henske</span>, <span>M. Theodorou</span>, <span>D. Valentine</span>, <span>M. O&rsquo;Malley</span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    August 2019
  </span>
  

  

  

  
  
  

  
  

  
    

  

</div>

  











  



<div class="btn-links mb-3">
  
  








  
    
  



<a class="btn btn-outline-primary my-1 mr-1" href="https://pubs.acs.org/doi/abs/10.1021/acssynbio.9b00271" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 js-cite-modal"
        data-filename="/publication/2019_acs_synbio/cite.bib">
  Cite
</button>















</div>


</div>


<div class="article-header article-container featured-image-wrapper mt-4 mb-4" style="max-width: 720px; max-height: 308px;">
  <div style="position: relative">
    <img src="/publication/2019_acs_synbio/featured_hu4b29ffa557ab88840f64c15ee8ee3a04_326347_720x0_resize_q90_lanczos.jpg" alt="" class="featured-image">
    <span class="article-header-caption">Anaerobic gut fungi degrade lignin.</span>
  </div>
</div>



  <div class="article-container">

    
    <h3>Abstract</h3>
    <p class="pub-abstract">Consortium-based approaches are a promising avenue toward efficient bioprocessing. However, many complex microbial interactions dictate community dynamics and stability that must be replicated in synthetic systems. The rumen and/or hindguts of large mammalian herbivores harbor complex communities of biomass-degrading fungi and bacteria, as well as archaea and protozoa that work collectively to degrade lignocellulose, yet the microbial interactions responsible for stability, resilience, and activity of the community remain largely uncharacterized. In this work, we demonstrate a “top-down” enrichment-based methodology for selecting a minimal but effective lignocellulose-degrading community that produces methane-rich fermentation gas (biogas). The resulting enrichment consortium produced 0.75–1.9-fold more fermentation gas at 1.4–2.1 times the rate compared to a monoculture of fungi from the enrichment. Metagenomic sequencing of the top-down enriched consortium revealed genomes encoding for functional compartmentalization of the community, spread across an anaerobic fungus (Piromyces), a bacterium (Sphaerochaeta), and two methanogenic archaea (Methanosphaera and Methanocorpusculum). Guided by the composition of the top-down enrichment, several synthetic cocultures were formed from the “bottom-up” using previously isolated fungi, Neocallimastix californiae and Anaeromyces robustus paired with the methanogen Methanobacterium bryantii. While cross-feeding occurred in synthetic co-cultures, removal of fungal metabolites by methanogens did not increase the rate of gas production or the rate of substrate deconstruction by the synthetic community relative to fungal monocultures. Metabolomic characterization verified that syntrophy was established within synthetic co-cultures, which generated methane at similar concentrations compared to the enriched consortium but lacked the temporal stability (resilience) seen in the native system. Taken together, deciphering the membership and metabolic potential of an enriched gut consortium enables the design of methanogenic synthetic co-cultures. However, differences in the growth rate and stability of enriched versus synthetic consortia underscore the difficulties in mimicking naturally occurring syntrophy in synthetic systems.</p>
    

    
    <div class="row">
      <div class="col-md-1"></div>
      <div class="col-md-10">
        <div class="row">
          <div class="col-12 col-md-3 pub-row-heading">Type</div>
          <div class="col-12 col-md-9">
            
            
            <a href="/publication/#2">
              Journal article
            </a>
            
          </div>
        </div>
      </div>
      <div class="col-md-1"></div>
    </div>
    <div class="d-md-none space-below"></div>
    

    
    <div class="row">
      <div class="col-md-1"></div>
      <div class="col-md-10">
        <div class="row">
          <div class="col-12 col-md-3 pub-row-heading">Publication</div>
          <div class="col-12 col-md-9"><em>ACS Synthetic Biology</em></div>
        </div>
      </div>
      <div class="col-md-1"></div>
    </div>
    <div class="d-md-none space-below"></div>
    

    <div class="space-below"></div>

    <div class="article-style"></div>

    

<div class="article-tags">
  
  <a class="badge badge-light" href=""></a>
  
</div>


    








  
  
    
  
  






  
  
  
  
  <div class="media author-card">
    

    <div class="media-body">
      <h5 class="card-title"><a href="/authors/s.-gilmore/"></a></h5>
      
      
      <ul class="network-icon" aria-hidden="true">
  
</ul>

    </div>
  </div>




    


  </div>
</div>



      

    
    

    
    
    
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.4/imagesloaded.pkgd.min.js" integrity="sha256-lqvxZrPLtfffUl2G/e7szqSvPBILGbwmsGE1MKlOi0Q=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.6/isotope.pkgd.min.js" integrity="sha256-CBrpuqrMhXwcLLUd5tvQ4euBHCdh7wGlDfNz8vbu/iI=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.js" integrity="sha256-X5PoE3KU5l+JcX+w09p/wHl9AzK333C4hJ2I9S5mD4M=" crossorigin="anonymous"></script>

      

      
        
        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.6/highlight.min.js" integrity="sha256-aYTdUrn6Ow1DDgh5JTc3aDGnnju48y/1c8s1dgkYPQ8=" crossorigin="anonymous"></script>
        
        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.6/languages/r.min.js"></script>
        
      

      
      
    

    
    
      <script src="https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.2.0/leaflet.js" integrity="sha512-lInM/apFSqyy1o6s89K4iQUKg6ppXEgsVxT35HbzUupEVRh2Eu9Wdl4tHj7dZO0s1uvplcYGmt3498TtHq+log==" crossorigin="anonymous"></script>
    

    
    
    <script>hljs.initHighlightingOnLoad();</script>
    

    
    
    
    
    
    
    <script>
      const search_config = {"indexURI":"/index.json","minLength":1,"threshold":0.3};
      const i18n = {"no_results":"No results found","placeholder":"Search...","results":"results found"};
      const content_type = {
        'post': "Posts",
        'project': "Projects",
        'publication' : "Publications",
        'talk' : "Talks"
        };
    </script>
    

    
    

    
    
    <script id="search-hit-fuse-template" type="text/x-template">
      <div class="search-hit" id="summary-{{key}}">
      <div class="search-hit-content">
        <div class="search-hit-name">
          <a href="{{relpermalink}}">{{title}}</a>
          <div class="article-metadata search-hit-type">{{type}}</div>
          <p class="search-hit-description">{{snippet}}</p>
        </div>
      </div>
      </div>
    </script>
    

    
    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/3.2.1/fuse.min.js" integrity="sha256-VzgmKYmhsGNNN4Ph1kMW+BjoYJM2jV5i4IlFoeZA9XI=" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/jquery.mark.min.js" integrity="sha256-4HLtjeVgH0eIB3aZ9mLYF6E8oU5chNdjU6p6rrXpl9U=" crossorigin="anonymous"></script>
    

    
    

    
    

    
    
    
    
    
    
    
    
    
      
    
    
    
    
    <script src="/js/academic.min.6cbefe3f0755301b86cd38317ded9f54.js"></script>

    






  
  <div class="container">
    <footer class="site-footer">
  
  <p class="powered-by">
    
      <a href="/privacy/">Privacy Policy</a>
    
    
       &middot; 
      <a href="/terms/">Terms</a>
    
  </p>
  

  <p class="powered-by">
    

    Powered by the
    <a href="https://sourcethemes.com/academic/" target="_blank" rel="noopener">Academic theme</a> for
    <a href="https://gohugo.io" target="_blank" rel="noopener">Hugo</a>.

    
    <span class="float-right" aria-hidden="true">
      <a href="#" class="back-to-top">
        <span class="button_icon">
          <i class="fas fa-chevron-up fa-2x"></i>
        </span>
      </a>
    </span>
    
  </p>
</footer>

  </div>
  

  
<div id="modal" class="modal fade" role="dialog">
  <div class="modal-dialog">
    <div class="modal-content">
      <div class="modal-header">
        <h5 class="modal-title">Cite</h5>
        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
          <span aria-hidden="true">&times;</span>
        </button>
      </div>
      <div class="modal-body">
        <pre><code class="tex hljs"></code></pre>
      </div>
      <div class="modal-footer">
        <a class="btn btn-outline-primary my-1 js-copy-cite" href="#" target="_blank">
          <i class="fas fa-copy"></i> Copy
        </a>
        <a class="btn btn-outline-primary my-1 js-download-cite" href="#" target="_blank">
          <i class="fas fa-download"></i> Download
        </a>
        <div id="modal-error"></div>
      </div>
    </div>
  </div>
</div>

</body>
</html>
